src/org/zaproxy/zap/extension/invoke/resources/help/contents/options.html

```diff
@@ -54,6 +54,16 @@ <H3>Parameters</H3>
 			<td>%postdata%</td>
 			<td>the POST data sent, if any, with any newlines replaced with "\n"</td>
 		</tr>
+		<tr>
+			<td>&nbsp;&nbsp;&nbsp;&nbsp;</td>
+			<td>%msgid%</td>
+			<td>the HTTP message id to fetch data the API (/JSON/core/view/message/?zapapiformat=JSON&id=$msgid)"\n"</td>

```

Comment:
- `the ID of the HTTP message to fetch data from the ZAP API`?
 - `zapapiformat=JSON&` can be removed (not used by the API).
 - `"\n"` should be removed.
 - Worth noting that the ID might not be always available.